src/ingestion/zengrc.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import base64
+import datetime
+import json
+import os
+
+from runners.helpers import db, log
+
+ZENGRC_ID = os.environ.get('ZENGRC_ID')
+ZENGRC_SECRET = os.environ.get('ZENGRC_SECRET')
+ZENGRC_URL = os.environ.get('ZENGRC_URL')
+ZENGRC_TABLE = os.environ.get('ZENGRC_TABLE')
+TIMESTAMP = str(datetime.datetime.utcnow())
+
+
+def gen_headers(id, secret):
+    auth = base64.b64encode(f'{id}:{secret}'.encode()).decode()
+    return {"Authorization": f"Basic {auth}"}
+
+
+def loop(endpoint):
```

Comment:
I would expect a function called `loop` to be really general, but here it seems relatively specific to the fact that you're using it to implement looping through pagination?. maybe `update_data` could work? in general functions should be named after the side-effects they have, where looping is an implementation detail.

Reply:
resolved